libcodechecker/libhandlers/check.py

```diff
@@ -452,12 +452,12 @@ def __update_if_key_exists(source, target, key):
                 quiet_build=args.quiet_build,
```

Comment:
I don't know if such exist. Most likely they don't, or if they were, they were moved to using `analyze` directly.

I don't know what or how could be added as an SA or Tidy args, so I'm unsure what could reliably made into a test case.